blog/_posts/2020-11-10-introduction-to-network-role.md

```diff
@@ -0,0 +1,298 @@
+---
+layout: post
+title: "Introduction to Network Role"
+section: Blog
+date: 2020-11-10T12:00:00
+author: Wen Liang
+category: talk
+---
+## Introduction
+
+The network role supports two providers: nm and initscripts. For CentOS/RHEL 6,
```

Comment:
"supports two providers: NetworkManager (nm) and initscripts."

The first time you introduce the abbreviation, spell it out so the reader knows what it is.  Then you can use `nm` in the rest of the document.